Groups/GroupAttendanceMulti.ascx.cs

```diff
@@ -612,6 +669,20 @@ protected void BindRepeat()
 
             lCount.Text = _attendees.Count( a => a.Attended ).ToString();
 
+            if ( lCount.Text != "0" )
```

Comment:
Consider just hiding the picker and showing the schedule literal when you are on an existing occurrence.

Reply:
Updated in latest commit, examples of how it shows now:

![image](https://github.com/user-attachments/assets/e6404861-6a93-4628-b80c-b307a35f137c)

![image](https://github.com/user-attachments/assets/b1fc7b95-c384-42e9-a0a0-d8d5107eb533)